pss-insight-data-sync.py

```diff
@@ -13,10 +14,16 @@
 source_url = os.getenv("SOURCE_URL")
 target_url = os.getenv("TARGET_URL")
 data_elements_url = os.getenv("SOURCE_DATA_ELEMENTS_URL")
+sync_check_url = urljoin(source_url.rstrip("/events"), "/datastore/configurations/sync_configs")
 
-# Check if any of the environment variables are missing
-if not (source_username and source_password and source_url and target_url):
-    raise ValueError("Please set the SOURCE_USERNAME, SOURCE_PASSWORD, SOURCE_URL, and TARGET_URL environment variables.")
+# Checking if sync_to_international attribute is set to True
+response_sync_check = requests.get(sync_check_url, auth=(source_username, source_password))
+
+if response_sync_check.status_code == 200 and response_sync_check.json() == True:
```

Comment:
The response from the above request looks like this:
```{
    "sync_to_international": true
}```

Please adjust this line accordingly.